File: Odd-Even Sort/OddEvenSort.js
const OddEvenSort = (arr) => {
  for (let i = 0; i < arr.length; ++i)
    for (let j = i % 2 == 0 ? 0 : 1; j + 1 < arr.length; j += 2)
      if (arr[j] > arr[j + 1]) [arr[j], arr[j + 1]] = [arr[j + 1], arr[j]];
  return arr;
};

File: Odd-Even Sort/OddEvenSort.cpp
void OddEvenSort(vector<int>& arr) {
    for (int i = 0; i < arr.size(); ++i)
        for (int j = ((i % 2 == 0) ? 0 : 1); j + 1 < arr.size(); j += 2)
            if (arr[j] > arr[j + 1])
                swap(arr[j], arr[j + 1]);
}